nullaway/src/main/java/com/uber/nullaway/NullabilityUtil.java

```diff
@@ -300,15 +300,15 @@ public static TreePath findEnclosingMethodOrLambdaOrInitializer(TreePath path) {
   /**
    * Check if a field might be null, based on the type.
    *
-   * @param symbol symbol for field
+   * @param symbol symbol for field; must be non-null
    * @param config NullAway config
    * @return true if based on the type, package, and name of the field, the analysis should assume
    *     the field might be null; false otherwise
+   * @throws NullPointerException if {@code symbol} is null
```

Comment:
We really should be running NullAway on NullAway 😅 

(Brings to mind a certain saying in Spanish about a blacksmith having only wooden tools)